src/components/homepage-features.tsx

```diff
@@ -28,7 +28,6 @@ const FeatureList: FeatureItem[] = [
         title: DataTypes.STRING,
         quantity: DataTypes.NUMBER,
       });
-
```

Comment:
Can you revert the changes in this file? I think the empty lines makes the examples easier to understand